app/src/main/java/com/swent/assos/ui/login/LoginScreen.kt

```diff
@@ -45,6 +46,7 @@ fun LoginScreen(navigationActions: NavigationActions) {
         var password by remember { mutableStateOf("") }
         val loginViewModel: LoginViewModel = hiltViewModel()
         var userNotFound by remember { mutableStateOf(false) }
+        var errorMessage by remember { mutableStateOf("") }
```

Comment:
Create the states `userNotFound` and  `errorMessage` in the `viewModel` and set its value directly in the method `signIn`

Reply:
Thank you ! I fixed it